gradle.properties

```diff
@@ -3,7 +3,8 @@ org.gradle.configureondemand=false
 org.gradle.parallel=true
 
 kotlin.code.style=official
+kotlin.mpp.enableGranularSourceSetsMetadata=true
+kotlin.mpp.enableCompatibilityMetadataVariant=true
+kotlin.native.enableDependencyPropagation=false
 
-android.useAndroidX=true
-android.enableJetifier=true
-android.databinding.enableV2=true
\ No newline at end of file
+android.useAndroidX=true
```

Comment:
# Workaround for Bintray treating .sha512 files as artifacts
# https://github.com/gradle/gradle/issues/11412
systemProp.org.gradle.internal.publish.checksums.insecure=true